src/app_engine/apprtc.py

```diff
@@ -119,7 +119,22 @@ def get_wss_parameters(request):
   ws_tls = request.get('wstls')
 
   if not ws_host_port_pair:
-    ws_host_port_pair = constants.WSS_HOST_PORT_PAIR
+    memcache_client = memcache.Client()
+    wss_host_status_list = memcache_client.get(
+        constants.WSS_HOST_STATUS_LIST_KEY)
+    # Return the first server in the list that is up, which is the wss host
+    # that has been up the longest.
+    if wss_host_status_list is not None:
+      wss_host_status_list = [host for host in wss_host_status_list
```

Comment:
Take a look at the latest commit and tell me what you think, I modified your suggestion slightly to use next() with a default to avoid the try/catch.